crates/sui-types/src/multisig.rs

```diff
@@ -344,8 +335,7 @@ impl ToFromBytes for MultiSig {
         }
         let mut multisig: MultiSig =
             bcs::from_bytes(&bytes[1..]).map_err(|_| FastCryptoError::InvalidSignature)?;
```

Comment:
I think this line still needs to be removed somehow as it's an anti-pattern (per @bmwill)

Either that or MultiSig::from_bytes must not be called in any impl of Deserialize

Reply:
not sure what the alternative should be, whats the recommended pattern?

also can be addressed in a follow up as this is used a few times in other places, not directly related to this pr.

Reply:
Correct, all uses of bcs from within a Deserialize impl should be removed